frontend/public/components/catalog-items.jsx

```diff
@@ -2,20 +2,23 @@ import * as React from 'react';
 import * as _ from 'lodash-es';
 import * as PropTypes from 'prop-types';
 import {Link} from 'react-router-dom';
+
 import {CatalogTileView} from 'patternfly-react-extensions/dist/esm/components/CatalogTileView';
 import {CatalogTile} from 'patternfly-react-extensions/dist/esm/components/CatalogTile';
 import {VerticalTabs} from 'patternfly-react-extensions/dist/esm/components/VerticalTabs';
-
+import {FilterSidePanel} from 'patternfly-react-extensions/dist/esm/components/FilterSidePanel';
+import {EmptyState} from 'patternfly-react/dist/esm/components/EmptyState';
+import FormControl from 'patternfly-react/dist/esm/components/Form/FormControl';
```

Comment:
Keep the patternfly-react imports with the other yarn dependencies with an empty before the console imports.

Reply:
fixed